nbgrader/preprocessors/clearsolutions.py

```diff
@@ -15,6 +15,7 @@ class ClearSolutions(NbGraderPreprocessor):
 
     code_stub = Dict(
         dict(python="# YOUR CODE HERE\nraise NotImplementedError()",
+             R="# YOUR CODE HERE\nfail()",
```

Comment:
Is this related to this PR ?

Reply:
Yes, it's there because `ps2/problem.ipynb` needs it currently.

I can separate all the R stuff into its own PR if you want?

Reply:
(But really this is a fairly small improvement, and I'm not sure why R wasn't listed here in the first place)